src/Goerli-DssSpell.t.sol

```diff
@@ -464,136 +464,4 @@ contract DssSpellTest is GoerliDssSpellTestBase {
 
         assertEq(vest.rxd(1), WAD);
     }
```

Comment:
Consider cleanup `testNewChainlogValues` and `testNewIlkRegistryValues` and set visibility to `private` to disable.

Reply:
Done in [7900c26](https://github.com/makerdao/spells-goerli/pull/104/commits/7900c2635a91d18e257ccc5e89e98ce865e05c6b)